nominees/a11y.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "a11yproject",
+    "description": "The A11Y Project is a community-driven effort to make digital accessibility easier.",
+    "website": "https://www.a11yproject.com/",
+    "license": [{
+        "spdx": "Apache-2.0",
+        "licenseURL": "https://github.com/a11yproject/a11yproject.com/blob/main/LICENSE-APLv2"
+    }],
+    "SDGs": [{
+            "SDGNumber": 8,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+        },
+        {
+            "SDGNumber": 9,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+
+        },
+        {
+            "SDGNumber": 12,
+            "evidenceURL": "https://www.a11yproject.com/about/"
+
+        }
+    ],
+    "type": [
```

Comment:
@SondosAhmed the Apache-2.0 license only caters for the `software` category as per the OSI list referenced [here](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md). Unless you have an explicit license for the `data` category from the above OSI [data category list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md), I would suggest you remove the data category.